core-web/libs/portlets/dot-ema/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -133,4 +149,34 @@ export class DotEmaShellComponent implements OnInit, OnDestroy {
         this.destroy$.next(true);
         this.destroy$.complete();
     }
+
+    resetDialogIframeData() {
+        this.store.resetDialog();
+    }
+
+    onIframeLoad() {
+        this.store.setDialogIframeLoading(false);
```

Comment:
this also can be in a `tap` inside the `pipe`

Reply:
I prefer to call it explicitly